patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,88 @@
+- Unsure if the tools used by the project and the organizational processes are conducive to InnerSource style development.
+
+## Sketch (optional)
+
+## Solutions
+
+This pattern presents a framework (_Project-Process-People_ model) and an _accompanying tool_ as one specific implementation of the framework for identifying the _fitness_ of an InnerSource project. The framework is based on prior work by [Klaas-Jan Stol and Brian Fitzgerald](https://ulir.ul.ie/bitstream/handle/10344/4443/Stol_2014_inner.pdf?sequence=2) and [Silona Bonewald](https://innersourcecommons.org/assets/files/InnerSourceChecklist.pdf), and should help facilitate the identification of the most appropriate candidate InnerSource projects and the assessment of InnerSource-readiness of candidate projects.
+
+Based on previous work, we hypothesize that the _fitness_ of a project can be measured by examining the characteristics of the project, the tools used and the organizational practices, and the readiness of the team in-charge, which we call the  _Project-Process-People model_. We suggest two sub-factors within each which overall provides __six__ dimensions to measure the fitness of candidate projects.
+
+### List of factors to assess InnerSource Fitness
+
+1. __Project Compatibility__
+    - _Seed Product_ -> Do the project characteristics make it suitable for InnerSource style development?
+    - _Multiple Potential Stakeholders_ -> Is the project likely to attract developers from different teams and build a community around the project?
+2. __Process Compatibility__
+    - _Appropriate Tools and Practices_ -> Do project practices facilitate contributions from outsiders and are the tools used in the project too specific and need to learned?
+    - _Ease of contribution_ -> Are the required ancillary resources set up and is the code modular enough to facilitate contribution?
+3. __People Compatibility__
+    - _Team Readiness for InnerSource_ -> Are the team members aware of and ready for the challenges of maintaining an InnerSource project?
+    - _Management Support_ -> Is the management willing to support the developers working on the project and the initiative in general?
+
+As one specific implementation of the pattern, we present a tool for estimating the _fitness_ of a candidate project along the six dimensions, and also present an an aggregate score which is a weighted sum of the scores for the dimensions (weights are determined by how many people voted a characteristic as important in the [2020 InnerSource Survey](https://tapjdey.github.io/InnerSource_Survey_2020/index.html)).
+The tool is written in __R__, and can be run from the [GitHub repository](https://github.com/tapjdey/InnerSource_Project_Fitness) (recommended - instructions in the repository) or from an [online instance](https://tapjdey.shinyapps.io/InnerSource_Fitness_Checker/). The tool is available in Chinese and English (language can be changed on the fly).
```

Comment:
```suggestion
## Reference Implementation

The [InnerSource Project Fitness Tracker](https://github.com/tapjdey/InnerSource_Project_Fitness) is one specific implementation of this pattern. It helps estimating the _fitness_ of a candidate project along the six dimensions explained above, and presents an aggregate score which is the weighted sum of the scores for these dimensions. (As an side: Weights are determined by how many people considered a characteristic important in the [InnerSource Survey 2020](https://tapjdey.github.io/InnerSource_Survey_2020/index.html)).
```

I would move the specifics of the tool to the GitHub repo instead. (thinks like the language, the translation, etc)

Also I recall that you had mentioned some limitations of the [online version](https://tapjdey.shinyapps.io/InnerSource_Fitness_Checker/)? The way it is hosted limits it to so-and-so many executions per month, right? Do you still want to add a link to the online version here?

In the context of the pattern I would focus on how the tool helps to solve the problem.